### PR TITLE
Seeding Script and API Route

### PR DESCRIPTION
**Pull request by inkaivasyuk**
- Set up server, api route, and database. Created seeding script.

- The file "scripts/seed.js" was mainly for practice. I was saving to my local folder. 
When I was able to make the script work I added to S3 and my database.

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-console */
```
Comment (pbennettwa) [inline database/index.js]: interesting, I'm not aware of this eslint config, I'm curious since mine isn't working

File: database/schema.sql [hunk 2]
```diff
@@ -0,0 +1,12 @@
+DROP DATABASE IF EXISTS photos;
```
Comment (pbennettwa) [inline database/schema.sql]: smart, I'm actually changing my mongoDB to only update tomorrow.  This seems like the right move.

File: scripts/check-aws-credentials.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+/* eslint-disable no-console */
+const AWS = require('aws-sdk');
```
Comment (pbennettwa) [inline scripts/check-aws-credentials.js]: fancy

File: scripts/seed.js [hunk 4]
```diff
@@ -1 +1,34 @@
...
+  }));
+};
+
+for (let i = 0; i < 100; i += 1) {
+  fs.mkdir(path.join(__dirname, `${dir}`, `${i}`), { recursive: true }, (err) => {
+    if (err) {
+      throw err;
+    } else {
+      console.log('Created new directory.');
+
```
Comment (pbennettwa) [inline scripts/seed.js]: I'm curious why many setTimeouts here are necessary.  This seems like you could do this using one of the same in a fetch using whatever you want to use on the client side.  I do get the reasoning though.

File: scripts/upload-to-aws-bucket.js [hunk 5]
```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
+const AWS = require('aws-sdk');
+const request = require('request');
+const db = require('../database/index');
+
```
Comment (pbennettwa) [inline scripts/upload-to-aws-bucket.js]: Cool stuff to learn, all of this is news to me.

File: server/index.js [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/photos', (req, res) => {
+  db.query('SELECT * FROM photos', (err, data) => {
+    if (err) {
+      console.log(err);
+      res.sendStatus(500);
+    } else {
```
Comment (pbennettwa) [inline server/index.js]: this makes sense since you have to pull a random amount of images.

File: database/index.js (hunk 1 above)
Comment (inkaivasyuk) [reply database/index.js]: Yeah, eslint tells me that I have unnecessary console.logs. It doesn't like them. I needed to keep some of the console.logs so I had to disable for those specific files so I don't see the annoying squiggly line from eslint lol

File: scripts/seed.js (hunk 4 above)
Comment (inkaivasyuk) [reply scripts/seed.js]: I was having trouble getting images from the api, the socket or port kept closing/dropping cause I think they couldn't handle me sending request for 600 images.

File: scripts/upload-to-aws-bucket.js (hunk 5 above)
Comment (inkaivasyuk) [reply scripts/upload-to-aws-bucket.js]: It was a bit of a challenge understanding their documents and how I really am supposed to save on S3, but it was pretty interesting in the end.

File: server/index.js (hunk 6 above)
Comment (inkaivasyuk) [reply server/index.js]: I made one to get all the images and one where I am looking for a certain 'house/Listing_id'

**Pull request merged**